packages/launcher/src/renderer/UIComponents/AlertToast.js

```diff
@@ -0,0 +1,67 @@
+//@flow
+
+import React, { Component } from 'react'
+import ReactModal from 'react-modal'
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group'
```

Comment:
Could you use https://github.com/reactjs/react-transition-group/tree/v1-stable#high-level-api-csstransitiongroup rather than `react-addons-css-transition-group` that seems no longer maintained please?